astropy/wcs/wcs.py

```diff
@@ -1898,7 +1898,7 @@ def all_world2pix(self, *args, **kwargs):
                HST's ACS/WFC detector, which has the strongest
                distortions of all HST instruments, testing has
                shown that enabling this option would lead to a about
-               50-100\% penalty in computational time (depending on
+               50-100\\% penalty in computational time (depending on
```

Comment:
`\%` is deprecated (the only allowed escaped sequences are listed in this [table](https://docs.python.org/2/reference/lexical_analysis.html#string-literals)) so it throws an Exception. That's why I escaped the first `\`.

Just remove (both) `\`?